custom-recipes/sentence-embedding-compute/recipe.py

```diff
@@ -78,7 +78,7 @@
         new_column_name += "_{}".format(j)
 
     # Adding a new column with computed embeddings
-    df[new_column_name] = map(json.dumps, embedded_texts)
+    df[new_column_name] = list(map(json.dumps, embedded_texts))
```

Comment:
For my curiosity, why is this done here? I'm assuming that it's something to do with python3 compat ?

Reply:
Py3 returns a map object, Py2 a list; casting to list works with both 👍